cilium/tox.ini

```
@@ -26,8 +26,11 @@ commands =
     pip install -r requirements.in
     pytest -v {posargs}
 setenv =
-    CILIUM_LEGACY=false
-    legacy: CILIUM_LEGACY=true
-    CILIUM_VERSION = 1.9.10
+    CILIUM_LEGACY = false
+    legacy: CILIUM_LEGACY = true
+    legacy: CILIUM_VERSION = 1.9.10
+    1.9: CILIUM_VERSION = 1.9.11
```

```suggestion
    1.9: CILIUM_VERSION = 1.9.10
```
we may want to test the same version on legacy non legacy